src/parser/demonhunter/havoc/modules/features/Checklist/Component.js

```diff
@@ -43,10 +43,40 @@ class HavocDemonHunterChecklist extends React.PureComponent {
         >
           {combatant.hasTalent(SPELLS.IMMOLATION_AURA_TALENT.id) && <AbilityRequirement spell={SPELLS.IMMOLATION_AURA_TALENT.id} />}
           {combatant.hasTalent(SPELLS.FELBLADE_TALENT.id) && <AbilityRequirement spell={SPELLS.FELBLADE_TALENT.id} />}
-          {combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) && <AbilityRequirement spell={SPELLS.BLADE_DANCE.id} />}
-          {combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) && <AbilityRequirement spell={SPELLS.DEATH_SWEEP.id} />}
+          {combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) && <AbilityRequirement spell={SPELLS.BLADE_DANCE.id} /> && <AbilityRequirement spell={SPELLS.DEATH_SWEEP.id} />}
```

Comment:
This does not work as only the first ability requirement would be rendered. What did you intend to do?

Reply:
Yeah i wasnt sure if it would do anything or not. But it seems it behaves in the way you said. Due to me combining the death sweep and blade dance abilities as 1 shared spell / cast efficiency in the abilities file i wasnt sure if i need to add both here or not. it seems i dont. so removing it.